TaskModules/powershell/TaskModuleIISManageUtility/AppCmdOnTargetMachines.ps1

```diff
@@ -731,6 +731,35 @@ function Start-Stop-Recycle-ApplicationPool {
     Invoke-VstsTool -Filename $appCmdPath -Arguments $appCmdArgs -RequireExitCodeZero
 }
 
+function ConfigureWebsiteAuthentication {
+    param (
+        [string]$basicAuthentication,

```

Comment:
Should we take one single parameter authentication type and use it? This method supports two types today, but extending to other types like (Forms or ASP.Net impersonation etc) in future might mean we change the function contract.